tests/lib/prepare-project.sh

```diff
@@ -32,6 +44,28 @@ build_deb(){
     cp ../*.deb "$GOHOME"
 }
 
+fedora_build_rpm() {
+    release=$(echo "$SPREAD_SYSTEM" | awk '{split($0,a,"-");print a[2]}')
+    arch=x86_64
+
+    base_version="$(head -1 debian/changelog | awk -F'[()]' '{print $2}')"
+    version="1337.$base_version"
```

Comment:
Yes, this version is picked to ensure its absurdly high to ensure we always have something more current installed than what is in the core snap.